src/features/events/components/EventParticipantsList.tsx

```diff
@@ -42,9 +43,26 @@ const EventParticipantsList: FC<EventParticipantsListProps> = ({
         description={messages.eventParticipantsList.descriptionBooked()}
         model={model}
         orgId={orgId}
-        rows={model.getParticipants().data ?? []}
+        rows={model.getBookedParticipants()}
         title={messages.eventParticipantsList.bookedParticipants()}
+        type="booked"
       />
+      {model.getNumCancelledParticipants() > 0 && (
+        <ParticipantListSection
+          chipColor={model.getParticipantStatus()}
```

Comment:
This should not be dynamically colored, but always be gray (like the signups section).

Reply:
Fixed.